multimodal/src/autogluon/multimodal/data/dataset.py

```diff
@@ -35,6 +36,9 @@ def __init__(
             A list of multimodal feature preprocessors generating model-agnostic features.
         processors
             Data processors customizing data for each modality per model.
+        id_mappings
+             Id-to-content mappings. The contents can be text, image, etc.
+             This is used when the dataframe contains the query/response indexes instead of their contents.
```

Comment:
I think str—> dict is fine for now. But we may consider to change it to str —> dataframe. Because we can just store all data in Dataframe-like objects.

Reply:
Also, we may need to revise the name, like calling it `linked_data`.

The idea is that the columns in the train_data / val_data are actually linked to another dataframe.

Reply:
Or "linked_train_data".